Spanner/src/Operation.php

```diff
@@ -83,8 +83,6 @@ public function __construct(ConnectionInterface $connection, $returnInt64AsObjec
      */
     public function mutation($operation, $table, $mutation)
     {
-        $mutation = $this->arrayFilterRemoveNull($mutation);
```

Comment:
Could you touch on the original intent of this? Are there side effects we should be concerned about as a result of its omission?

Reply:
The original intent is no longer clear to my memory. I ran the system test suite and also did fairly extensive tests on my own and do not believe there are any negative side effects to this change.

Reply:
I took a look as well, the fix seems reasonable.